### PR TITLE
Update phonenumberutiltest.py

### PR DESCRIPTION
**Pull request by penmetsaa**
Based on changes suggested in cl/451855712 by pylint

### CODE REVIEW

File: python/tests/phonenumberutiltest.py [hunk 1]
```diff
@@ -2978,19 +2978,19 @@ def testIsMobileNumberPortableRegion(self):
         self.assertFalse(phonenumbers.is_mobile_number_portable_region("XY"))
 
     def testGetMetadataForRegionForNonGeoEntity_shouldBeNull(self):
-        self.assertTrue(PhoneMetadata.metadata_for_region("001") is None)
+        self.assertIs(PhoneMetadata.metadata_for_region("001"), None)
```
Comment (daviddrysdale) [inline python/tests/phonenumberutiltest.py]: (Note to self: this [requires Python 2.7](https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertIs), which is probably OK at this point &hellip;)

LGTM, just a couple of things:
- Could you apply the same change to all of the `self.assertTrue(... is None)` instances?  I think there's another dozen or so.
- Maybe shift to `self.assertIsNone` for better clarity?

Comment (penmetsaa) [reply python/tests/phonenumberutiltest.py]: Done David. Did only in this test file. Do you want me to extend to other files as well for consistency?
PTAL and suggest.

**Review by daviddrysdale — APPROVED** [review]

**Comment by AA-Turner** [comment]
@daviddrysdale -- are you also bumping the package minimum to 2.7, or only test requirements? 

If the former I'm happy to prepare a PR updating to 2.7.

A

**Comment by daviddrysdale** [comment]
Thanks, but I'm not aware of anything (non-test) that would *break* for 2.5 & 2.6, so I don't propose to change anything.

**Comment by AA-Turner** [comment]
> Thanks, but I'm not aware of anything (non-test) that would break for 2.5 & 2.6, so I don't propose to change anything.

No worries -- it's much easier doing no work!

A

**Pull request merged**